packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -1070,6 +1112,24 @@ export class CrossChainMessenger implements ICrossChainMessenger {
       )
     },
 
+    approveERC20: async (
+      l1Token: AddressLike,
+      l2Token: AddressLike,
+      amount: NumberLike,
+      opts?: {
+        overrides?: Overrides
+      }
+    ): Promise<BigNumber> => {
+      return this.l1Provider.estimateGas(
+        await this.populateTransaction.approveERC20(
+          l1Token,
+          l2Token,
+          amount,
+          opts
```

Comment:
Should this be `opts?.overrides` ?

Reply:
Nah, this gets passed to a `IBridgeAdapter` function which expects `opts` to be passed in. It's not being passed to a function on an `ethers.Contract` object.